src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/AddConsoleCommandPassTest.php

```diff
@@ -36,24 +36,7 @@ public function testProcess()
         $this->assertSame('my-command', (string) $container->getAlias($alias));
 
         $this->assertTrue($container->hasParameter('console.command.ids'));
-        $this->assertSame(array('my-command'), $container->getParameter('console.command.ids'));
-    }
-
-    /**
-     * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage The service "my-command" tagged "console.command" must be public.
-     */
-    public function testProcessThrowAnExceptionIfTheServiceIsNotPublic()
-    {
-        $container = new ContainerBuilder();
-        $container->addCompilerPass(new AddConsoleCommandPass());
-
-        $definition = new Definition('Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler\MyCommand');
-        $definition->addTag('console.command');
-        $definition->setPublic(false);
```

Comment:
Maybe I'm missing something but, shouldn't you keep the `$definition->setPublic(false);` line to test that private commands don't trigger an error?